src/components/select-box/_select-box.scss

```diff
@@ -0,0 +1,28 @@
+@import "../../globals/scss/import-once";
+@import "../../globals/scss/colours";
+@import "../../globals/scss/typography";
+
+@include exports("select-box") {
+  .govuk-c-select-box {
+    box-sizing: border-box; // should this be global?
+    font-family: $govuk-font-stack;
+    -webkit-font-smoothing: antialiased;
```

Comment:
can we use the font-smoothing mixin here instead please? - @include font-smoothing